scripts/checkpatch_inc.sh

```diff
@@ -7,9 +7,14 @@ CHECKPATCH_IGNORE=$(echo core/lib/lib{fdt,tomcrypt} lib/lib{png,utils,zlib} \
 _CP_EXCL=$(for p in $CHECKPATCH_IGNORE; do echo ":(exclude)$p" ; done)
 
 function _checkpatch() {
+		# Use typedefsfile if supported by the checkpatch scripts
+		a="--typedefsfile typedefs.checkpatch"
```

Comment:
Do you think the trick `checkpatch.pl <some-supported-or-not-options> --help` is reliable in the "long term", or even clean ? I found no other ways to test option availability.